frontend/js/components/HistoryTable/HistoryTable.component.js

```diff
@@ -0,0 +1,70 @@
+// import '../../components/HistoryTable/HistoryTableRow.js'
+import Actions from '../../services/Action.service.js';
+
+class HistoryTable extends HTMLElement {
+
+    
+     constructor() {
+        super();
+        this.sessionsData = Actions.getAllSessions();
+    }
+
+    connectedCallback() {
+        this.render();
+        for(let i=0;i<this.sessionsData.length;i++)
+        {
+          document.getElementById(`${i}`).addEventListener('click',function(){ Actions.resumeSession(i)});
```

Comment:
dont need to use `${i}`, just use i. use ES6 notation for functions, let the definition of the function be in the next line with the right indentation

Reply:
fixed.